_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -228,10 +234,35 @@ This element is neither an `img` element nor has a role of `img`.
 <div aria-label="W3C logo"></div>
```

Comment:
Would it be worth adding the `background-image` property for consistency?

Reply:
Yaknow what, I don't think this is a good example. Putting aria-label on a random `div` does very little. I'm going to take this out.